release/release.go

```diff
@@ -627,6 +658,19 @@ func copyFile(src, dst string) error {
 	return out.Close()
 }
 
+func downloadFile(url, dst string) {
+	out, err := os.Create(dst)
```

Comment:
Do we really want to download files via HTTP instead of using the aws interface, now that we are importing it? To me, the answer to that would be whether or not there is a parallel downloader in the golang library.